src/components/step-sequence/step-sequence.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './step-sequence.scss';
+import './step-sequence-orientation.scss';
+
+const baseClass = 'carbon-step-sequence';
+const classes = orientation => `${baseClass} ${baseClass}--${orientation}`;
+
+const StepSequence = ({ children, orientation }) => (
+  <ol className={ classes(orientation) }>
```

Comment:
It might be worth adding an `aria-label` attribute on here, saying something like "step 3 of 5: delivery details".

Reply:
How would this work with flexible step labels,

For example, steps might be labeled `a` to `e`.

Would I just use an index and step count for this, regardless of the indicator used by the developer?

Reply:
Yes, I think that should work 👍

Reply:
The label will be hard now that it's not just a string, should I drop this if it's not a string?

Reply:
Maybe just truncate it to 'step 3 of 5'?

Reply:
Decided to put the label in as it's own prop rather than deriving it from other props

Reply:
Hmm, I'd initially meant for this to go on the `<ol>` element, but adding it to each step might be better for when the step contains complicated, non-string children.

It the steps do contain just strings though you probably won't need the `aria-label`, as you'd be duplicating the content. So these could probably be optional props instead?

Also, have a look at https://www.w3.org/WAI/tutorials/forms/multi-page/#using-step-by-step-indicator - it'd be good if we can add visually hidden 'completed' and 'current' indicators too.